data/preprocessing/input_preprocessing.py

```diff
@@ -121,13 +121,15 @@ def preprocess_image_and_label(image,
                                crop_width,
                                prev_image=None,
                                prev_label=None,
+                               depth=None,
```

Comment:
Did you also update data/preprocessing/input_preprocessing_test.py?

Reply:
Thanks for the comment! I have updated the test script.